kernel/src/process.rs

```diff
@@ -612,7 +621,7 @@ pub trait Process {
         driver_num: usize,
         size: usize,
         align: usize,
-    ) -> bool;
+    ) -> Result<(), ()>;
```

Comment:
`Result` is the Rust way of signaling a function may fail. Additionally, `Result` triggers a warning when the return value is unused. From my little experiments, the new signature is compiled to the same code as the old one.